avaframe/in3Utils/cfgHandling.py

```diff
@@ -449,6 +449,10 @@ def applyCfgOverride(cfgToOverride, cfgWithOverrideParameters, module, addModVal
     overrideParameters = cfgWithOverrideParameters['%s_override' % modName]
     overrideKeys = [item for item in overrideParameters]
     overrideKeys.remove('defaultConfig')
+    if len(overrideKeys) != len(list(set(overrideKeys))):
+        message = 'duplicate parameter names appearing in override section'
+        log.error(message)
+        raise AssertionError
```

Comment:
This could be refactored to a function and be reused below, depending on whether there are actually two different error messages necessary?!

Reply:
it is two different scenarios: 1) in the override section itself a parameter has been used twice, 2) in the override section only one parameter was provided but this name appears twice in different sections of the module cfg where the override will be applied - I made a function for the error and checking for duplicate

Reply:
Jup, 2 scenarios but basically the same message: duplicate keys. Not sure it needs different messages?

Reply:
but once the duplicate keys appear in the override section and once for a key in the override section (also if appearing only once there) multiple exist in the config of the to override module so I thought it is helpful to know what is the issue